SpatialGDK/Source/SpatialGDK/Private/Interop/ActorSystem.cpp

```diff
@@ -1712,12 +1777,349 @@ FString ActorSystem::GetObjectNameFromRepState(const FSpatialObjectRepState& Rep
 	return TEXT("<unknown>");
 }
 
-bool ActorSystem::EntityHasComponent(const Worker_EntityId EntityId, const Worker_ComponentId ComponentId) const
+void ActorSystem::SendCreateEntityRequest(USpatialActorChannel* Channel, uint32& OutBytesWritten)
+{
+	UE_LOG(LogActorSystem, Log, TEXT("Sending create entity request for %s with EntityId %lld, HasAuthority: %d"),
+		   *Channel->Actor->GetName(), Channel->GetEntityId(), Channel->Actor->HasAuthority());
+
+	const Worker_RequestId RequestId = CreateEntity(Channel, OutBytesWritten);
+
+	NetDriver->Receiver->AddPendingActorRequest(RequestId, Channel);
```

Comment:
Shouldn't these pending requests be handled by the actor system now?

Reply:
These are related to commands- so will be moved over to the Actor System as of https://improbableio.atlassian.net/browse/UNR-4848